UserContext/AnonymousRequestMatcher.php

```diff
@@ -0,0 +1,62 @@
+<?php
+/*
+ * This file is part of the FOSHttpCacheBundle package.
+ *
+ * (c) FriendsOfSymfony <http://friendsofsymfony.github.com/>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace FOS\HttpCacheBundle\UserContext;
+
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\RequestMatcherInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+class AnonymousRequestMatcher implements RequestMatcherInterface
```

Comment:
please add a class level comment to explain that this matcher matches all requests that are anonymous.